python_modules/dagster/dagster/core/definitions/decorators/repository_decorator.py

```diff
@@ -19,6 +20,30 @@
 from ..unresolved_asset_job_definition import UnresolvedAssetJobDefinition
 
 
+def _flatten(items):
+    from dagster.core.asset_defs import AssetGroup, AssetsDefinition, SourceAsset
+
+    BaseTypes = (
```

Comment:
It might be cumbersome to remember to include new definitions here when they're added. Thoughts on doing it the other way around? I.e. checking to see if the item is _not_ a list?

Reply:
yeah, could be more strict about it and not allow tuples or whatever else people come up with